packages/mobile/src/onboarding/registration/NameAndPicture.test.tsx

```diff
@@ -14,26 +14,6 @@ expect.extend({ toBeDisabled })
 const mockScreenProps = getMockStackScreenProps(Screens.NameAndPicture)
 
 describe('NameAndPictureScreen', () => {
-  it('renders correctly', () => {
-    const store = createMockStore()
-    const tree = render(
-      <Provider store={store}>
-        <NameAndPicture {...mockScreenProps} />
-      </Provider>
-    )
-    expect(tree).toMatchSnapshot()
-  })
-
-  it('renders with an error', () => {
-    const store = createMockStore({ alert: { underlyingError: ErrorMessages.INVALID_INVITATION } })
-    const tree = render(
-      <Provider store={store}>
-        <NameAndPicture {...mockScreenProps} />
-      </Provider>
-    )
-    expect(tree).toMatchSnapshot()
-  })
-
```

Comment:
These tests didn't really add any value. The snapshots for each are the same and subsequent tests verify the actual functionality